### PR TITLE
add  ul/li  buttonlist

### PR DESCRIPTION
**Pull request by HorstFrank**

### CODE REVIEW

File: components/QuestionButtonset/QuestionButtonset.tsx [hunk 1]
```diff
@@ -1,17 +1,18 @@
 import React from "react";
-import DummyButton from "../Button/DummyButton";
+
 import "./QuestionButtonset.module.css";
 // import "../../styles/globals.css";
 // import { ButtonType } from "../../types/ButtonType";
```
Comment (mewiemers) [inline components/QuestionButtonset/QuestionButtonset.tsx]: do you need this anymore?

**Review by mewiemers — COMMENTED** [review]
very good job! 🎉🎉

**Review by m-wiemers — APPROVED** [review]
Mega awesome great!

**Review by mrana17 — APPROVED** [review]
Lgtm

**Review by RonjaMeinecke — APPROVED** [review]
💥LGTM

**Pull request merged**